app/controllers/sign_in/client_configs_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SignIn
+  class ClientConfigsController < SignIn::ApplicationController
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
+    skip_before_action :authenticate
+    before_action :authenticate_service_account
+    before_action :set_client_config, only: %i[show update destroy]
+
+    def index
+      client_configs = if params[:client_ids].presence
+                         SignIn::ClientConfig.where(client_id: params[:client_ids])
+                       else
+                         SignIn::ClientConfig.all
+                       end
+
+      render json: client_configs, status: :ok
+    end
+
+    def show
+      render json: @client_config, status: :ok
+    end
+
+    def create
+      client_config = SignIn::ClientConfig.new(client_config_params)
+
+      if client_config.save
+        render json: client_config, status: :created
+      else
+        render json: client_config.errors, status: :unprocessable_entity
+      end
+    end
+
+    def update
+      if @client_config.update(client_config_params)
```

Comment:
Every time I see mass assignment I get so suspicious and it freaks me out but I guess this is like normal and even 'recommended' rails so I guess I won't block it

Reply:
Yeah, I think it's safe as long as strong parameters are used